qt_dotgraph/src/qt_dotgraph/dot_to_qt.py

```diff
@@ -252,7 +255,7 @@ def dotcode_to_qt_items(self, dotcode, highlight_level, same_label_siblings=Fals
             graph = graph[0]
 
         #graph = pygraphviz.AGraph(string=self._current_dotcode, strict=False, directed=True)
-        #graph.layout(prog='dot')
+        # graph.layout(prog='dot')
```

Comment:
commented code

Reply:
The goal of these changes is to get the code to pass autopep8 the same way we style check the MoveIt! project using clang formatting. I agree that there is additional cleanup necessary but perhaps this may be beyond the scope of this effort. I will make the changes you suggested for this repo but I'm worried that we may get too far behind schedule cleaning up the ROS1 branches.

Reply:
Perhaps then don't remove commented code, but a lot of the line breaking stuff kinda made things worse.